Sources/MapboxMaps/Viewport/States/Overview/OverviewViewportStateOptions.swift

```diff
@@ -0,0 +1,22 @@
+import Turf
+import CoreLocation
+
+public struct OverviewViewportStateOptions: Equatable {
```

Comment:
This struct is merely Equatable and not Hashable since we didn't make Geometry Hashable in Turf v2